src/Microsoft.DotNet.Interactive/Parsing/DirectiveNode.cs

```diff
@@ -382,16 +382,19 @@ void WriteProperty(string propertyName, string value)
                 {
                     var bindingResult = await bind(expressionNode);
 
-                    if (bindingResult.IsSuccessful)
+                    if (bindingResult is not null)
```

Comment:
When can this return `null`? Should an error be reported (or an exception thrown) when `null` is returned?

Reply:
Maybe it can be modeled differently but we don't want to perform binding during language service commands, since the resulting directive command won't actually be invoked.